package.json

```diff
@@ -1,16 +0,0 @@
-{
-  "name": "restapi",
-  "version": "1.0.0",
-  "description": "",
-  "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "keywords": [],
-  "author": "",
-  "license": "ISC",
-  "dependencies": {
```

Comment:
What are these dependencies?
How do you find out if you need some specific dependency? For example: to fetch data from some cloud or google